.gitignore

```diff
@@ -2,3 +2,7 @@
 .vscode
 /.envrc
 .idea
+.nlsp-settings
```

Comment:
More up to @gakonst, but personally I just do `/.*` on my .gitignores and force-add whatever needs to be there (like .gitignore :sweat_smile:)

Reply:
Fine with me.as is!